tromega/lib/views/account/questions_widget.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:flutter/material.dart';
+import 'package:tromega/views/account/add_my_data_view.dart';
+import 'package:tromega/views/account/my_profile_view.dart';
+
+class FirstRouteWidget extends StatefulWidget {
+  const FirstRouteWidget({Key? key, this.title}) : super(key: key);
+
+  final String? title;
+
+  @override
+  _FirstRouteWidget createState() => _FirstRouteWidget();
+}
+
+class _FirstRouteWidget extends State<FirstRouteWidget> {
+  @override
+  Widget build(BuildContext context) {
+    final color = Theme.of(context).primaryColor;
+    return Padding(
+      padding: const EdgeInsets.all(15),
+      child: Center(
+          child: Column(
+        mainAxisAlignment: MainAxisAlignment.start,
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          buildQuestion(text: 'Wie alt bist du?'),
+          buildTextField(),
+          buildQuestion(text: 'Wie groß bist du?'),
+          buildTextField(),
+          buildQuestion(text: 'Wie viel wiegst du?'),
+          buildTextField(),
+          buildQuestion(text: 'Was ist dein Trainingsziel?'),
+          const SizedBox(height: 16),
+          buildDropdownButton(color,
+              dropDownOptions: ["Muskeln aufbauen", "Ausdauer verbessern", "Gewicht verlieren"],
+              dropdownValue: "Muskeln aufbauen"),
+          const SizedBox(height: 16),
+          buildQuestion(text: 'Mit welchem Geschlecht identifizierst du dich?'),
+          const SizedBox(height: 16),
+          buildDropdownButton(color,
+              dropDownOptions: ["männlich", "weiblich", "divers"],
+              dropdownValue: "männlich"),
+          const SizedBox(height: 32),
+          buildRouteButton(color),
+        ],
+      )),
+    );
+  }
+
+  Widget buildRouteButton(Color color) => Container(
+        height: 55,
+        padding: const EdgeInsets.only(left: 10, right: 10),
+        child: ElevatedButton(
+          style: ButtonStyle(
+              backgroundColor: MaterialStatePropertyAll<Color>(color)),
+          onPressed: () {
+            Navigator.push(
+              context,
+              MaterialPageRoute(builder: (context) => const SecondDataView()),
+            );
+          },
+          child: const Text('Weiter'),
+        ),
+      );
```

Comment:
So widgets kannst du auch gerne in eigene Dateien als custom widgets auslagern... muss jetzt aber noch nicht unbedingt